modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx

```diff
@@ -233,6 +233,10 @@ export function useObjectFieldForm({
 
 		const settings = normalizeFieldSettings(field.objectFieldSettings);
 
+		const maxUploadRequestSize = Math.floor(
+			Liferay.PropsValues.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE / 1000000
```

Comment:
@carolmariaabb 

If **1048576** is used instead of `Liferay.PropsValues.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE`, if the user manually updates the value for Overall Maximum Upload Request Size in the system settings, the value is not updated in the attachment field validation. If `Liferay.PropsValues.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE` is used here, this value is updated and applied to the max file size validation. 

Please let me know which value you would like me to use, and I will update it along with the relocated line and added language key. Thank you!

Reply:
@SelenaAungst, I mean 1048576 instead of 1000000.

```
Liferay.PropsValues.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE / 1048576
```

Reply:
Ah I see, my apologies! I will have these changes updated ASAP. Thank you!

Reply:
@carolmariaabb 
I created the new pull request here: https://github.com/mbowerman/liferay-portal/pull/112. Thanks so much!

Reply:
I've force-pushed the changes to this pull request.